openslides_backend/presenter/base.py

```diff
@@ -0,0 +1,13 @@
+presenters = {}
```

Comment:
Please add typing and docstrings like in actions/actions.py and actions/base.py. Of course we can discuss whether to have actions/actions.py and base.py or only one of these files. We should do this for presenter equally. We can have a phone call if this is easier.

Reply:
See https://github.com/OpenSlides/openslides-backend/pull/40/commits/37d88fca9da06527257d44eea8bebf66562f94a0#diff-45ecc7a634e3cdfde96d2afe26f4af0d